contributors/join-us/recruiting.md

```diff
@@ -93,6 +93,10 @@ We can hire from many countries and regions across the globe. While in-office in
 
 ## Interview Stages
 
+As a fully remote and global team, Mattermost interviewers are located in different timezones. Since we also interview candidates across different time zones, we schedule interviews one-by-one vs. as a panel of interviews on one specific date. Below are the typical stages and amount of interviewers a candidate can expect in a full interview process at Mattermost. 
```

Comment:
```suggestion
As a fully remote and global team, Mattermost interviewers are located in different time zones. Since we also interview candidates across different time zones, we schedule interviews one-on-one as opposed to a panel of interviews on one specific date. Below are the typical stages and number of interviewers a candidate can expect in a full interview process at Mattermost.
```